code/modules/reagents/chemistry/reagents/alcohol.dm

```diff
@@ -470,7 +480,7 @@
 	drink_icon = "toxinsspecialglass"
 	drink_name = "Toxins Special"
 	drink_desc = "Whoah, this thing is on FIRE"
-	taste_message = "FIRE"
+	taste_message = "<span class='FIRE'>evil</span>"
```

Comment:
Uh. Do we have a `<span>` class called `FIRE`?...

Reply:
OOPS. My bad, can't believe I messed that one up.